ApollosProject/apollos-apps
# feat: Media Card

## ✏️ Description

This PR introduces the Media Card into the UI Kit and migrates the Progress Bar into UI Kit 2.0 as well

[Media Card Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=2678%3A21023&t=uSPjAx9dizhmj5iC-4)
[Progress Bar Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=2357%3A12468&t=uSPjAx9dizhmj5iC-4)

## 🔬 To Test
1. Set your PROJECT_VARIANT to `storybook`
2. Run storybook and check out the stories

## 📸 Screenshots

| Feature | Simulator | Figma | Links |
| --- | --- | --- | --- |
| Plain | ![Screenshot_20221221-131946](https://user-images.githubusercontent.com/45076058/208976468-684dd7f3-659d-460e-b687-4f6c9d9f1ec4.png) | ![Media Card](https://user-images.githubusercontent.com/45076058/208977039-f17c0ea8-c2eb-46a1-ba54-e5534e59ef35.png) | [Link to Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=2716%3A23487&t=uSPjAx9dizhmj5iC-4) |
| With Label | ![Screenshot_20221221-131857](https://user-images.githubusercontent.com/45076058/208976497-5558f454-afb6-4b40-beb2-5cd9cd21450b.png) | ![Media Card-1](https://user-images.githubusercontent.com/45076058/208977057-95aa7f17-951c-43f7-9cd8-7d6d5076d271.png) | [Link to Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=2716%3A23487&t=uSPjAx9dizhmj5iC-4) | The reason that we chose to do... |
| With Progress Bar | ![Screenshot_20221221-131919](https://user-images.githubusercontent.com/45076058/208976519-c99e1b00-3214-4555-84fc-e6c57ea2fb35.png) | ![Media Card-2](https://user-images.githubusercontent.com/45076058/208977123-5bd2211b-f376-4706-9376-080018155ba8.png) | [Link to Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=2716%3A23487&t=uSPjAx9dizhmj5iC-4) | The reason that we chose to do... |
| With Label and Progress Bar | ![Screenshot_20221221-131929](https://user-images.githubusercontent.com/45076058/208976534-edc73fd6-0af6-4ca7-82a9-0150d5c0b389.png) | ![Media Card-3](https://user-images.githubusercontent.com/45076058/208977148-4fe3b3b4-8290-4380-9ac8-e468a3fce243.png) | [Link to Figma](linkurl) | The reason that we chose to do... |

## ⚠️ To-do Before Merge



## 📝 Checklist
